### ISSUE TITLE
shift click on timeline should select regions

### ISSUE THREAD

**Issue by gfrogat** [Caleydo/malevo#43]

**Comment by NMO13** [comment]
@gfrogat pleas always write a short summary (or better provide screenshots) for issues. I know its pretty obvious in this case what you mean but it avoids confusion in every case.

**Comment by gfrogat** [comment]
@NMO13 is this possible with the new multi-dataset timeline? Can this issue be closed?

**Comment by thinkh** [comment]
No, that doesn't work. The d3 brush doesn't support this functionality AFAIK.

**Comment by NMO13** [comment]
I think its not top priority. But surely a nice to have.
